modules/javafx.graphics/src/main/java/javafx/scene/paint/RadialGradient.java

```diff
@@ -234,17 +234,21 @@ public final List<Stop> getStops() {
 
     /**
      * Creates a new instance of RadialGradient.
+     *
      * @param focusAngle the angle in degrees from the center of the gradient
-     * to the focus point to which the first color is mapped
+     *                   to the focus point to which the first color is mapped
      * @param focusDistance the distance from the center of the gradient to the
-     * focus point to which the first color is mapped
+     *                      focus point to which the first color is mapped,
+     *                      must be greater than or equal to zero
      * @param centerX the X coordinate of the center point of the gradient's circle
      * @param centerY the Y coordinate of the center point of the gradient's circle
-     * @param radius the radius of the circle defining the extents of the color gradient
+     * @param radius the radius of the circle defining the extents of the color gradient,
+     *               must be greater than or equal to zero
      * @param proportional whether the coordinates and sizes are proportional
-     * to the shape which this gradient fills
+     *                     to the shape which this gradient fills
      * @param cycleMethod cycle method applied to the gradient
      * @param stops the gradient's color specification
+     * @throws IllegalArgumentException if {@code focusDistance} or {@code radius} is negative
```

Comment:
can we use the `@since` tag applicable to `@throws` only?
(also below)

Reply:
I don't think this is possible.